### DISCUSSION TITLE
Multiple Rules Modifier

### DISCUSSION THREAD

**Discussion by Meldrak** [AdguardTeam/AdGuardHome#2852] [Q&A] [answered]
Hi,

Not sure it's a bug, but I want to use 2 rules modifiers for a domain and it's not working

`||my.site.org^$dnsrewrite=192.168.0.5^$client=192.168.0.0/24`

the goal is to rewrite a dns entry only if connected to local network

is the syntax incorrect or is it not supported?

**Comment by ameshkov** [comment, marked as answer]
You should use a comma to separate different modifiers. Please take a look here:
https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#modifiers

`||example.org^$dnsrewrite=192.168.0.5,client=192.168.0.0/24 `

**Comment by Meldrak** [comment]
Thanks it's working well, I missed this point on the wiki
